firstvoices/backend/views/category_views.py

```diff
@@ -172,46 +175,76 @@ def get_list_queryset(self):
             if len(flag)
         ]
 
-        # Check for the nested flag
-        nested_flag_input = self.request.GET.get("nested", True)
-        nested_flag = get_valid_boolean(nested_flag_input)
-
+        flags_query = Q()
         if len(contains_flags) > 0:
+            flags_query.connector = Q.OR
```

Comment:
just wondering, how this connector is used ? as I see that a connector is explicitly specified on line 184 as well for flags_query.

Reply:
The connector is added there because behaviour seems to be different if it is left with the default AND connector. I was not getting the expected result without it. See https://stackoverflow.com/questions/25066894/django-q-add-weird-behavior-when-using-operator-or for more info.

Reply:
Got it. Thanks